### PR TITLE
Corrected prod version set in Nightly

### PR DESCRIPTION
**Pull request by radtriste**
to respect current version standard

### CODE REVIEW

File: Jenkinsfile.nightly [hunk 1]
```diff
@@ -143,7 +143,7 @@ def getBuildConfigMap(def buildConfigPathFolder, def buildConfigAdditionalVariab
 def buildProject(String project, Map<String, Object> buildConfig, String defaultGroup = "kiegroup") {
     println "[INFO] Building project ${project}"
     if ("kiegroup/kogito-tooling" == project) {
-        env.CURRENT_KOGITO_TOOLING_VERSION = env.CURRENT_KOGITO_TOOLING_VERSION ?: sh(returnStdout: true, script: 'awk -F\'"\' \'/"version": ".+"/{ print $4; exit; }\' lerna.json')?.trim() + "-redhat-${env.DATE_TIME_SUFFIX}"
+        env.CURRENT_KOGITO_TOOLING_VERSION = env.CURRENT_KOGITO_TOOLING_VERSION ?: sh(returnStdout: true, script: 'awk -F\'"\' \'/"version": ".+"/{ print $4; exit; }\' lerna.json')?.trim() + ".redhat-${env.DATE_TIME_SUFFIX}"
```
Comment (tiagobento) [inline Jenkinsfile.nightly:146]:
```suggestion
        env.CURRENT_KOGITO_TOOLING_VERSION = env.CURRENT_KOGITO_TOOLING_VERSION ?: sh(returnStdout: true, script: 'awk -F\'"\' \'/"version": ".+"/{ print $4; exit; }\' lerna.json')?.trim() + "-redhat+${env.DATE_TIME_SUFFIX}"
```

**Review by tiagobento — CHANGES_REQUESTED** [review]
Hi @radtriste. Thanks for the PR. I believe the changes you're proposing are not aligned with the SemVer specification. See https://semver.org/#spec-item-9

Also, I'm suggesting replacing the second `-` by a `+`, as SemVer specifies as well on the Build metadata part. See https://semver.org/#spec-item-10.

Prereleases should be sequential, as build metadata has a much more flexible format.

File: Jenkinsfile.nightly (hunk 1 above)
Comment (radtriste) [reply Jenkinsfile.nightly:146]: our main problem is that all version in with redhat suffix (RHPAM/RHDM/kogito runtimes) have the suffix with a `.` and not `-`.
This is causing problems on QE side due to that change ...
cc @jstastny-cz @mareknovotny @Ginxo

Comment (Ginxo) [reply Jenkinsfile.nightly:146]: @radtriste check
```
$ yarn lerna version 0.0.0.redhat-20210817 --no-git-tag-version --yes
yarn run v1.22.10
ERR! lerna bump must be an explicit version string _or_ one of: 'major', 'minor', 'patch', 'premajor', 'preminor', 'prepatch', or 'prerelease'.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
$ yarn lerna version 0.0.0-redhat-20210817 --no-git-tag-version --yes
yarn run v1.22.10
lerna notice cli v4.0.0
lerna info current version 0.0.0
lerna notice FYI git repository validation has been skipped, please ensure your version bumps are correct
lerna info Assuming all packages changed
lerna WARN version Skipping working tree validation, proceed at your own risk

Changes:
 - @kogito-tooling/backend-extended-services: 0.0.0 => 0.0.0-redhat-20210817
 - @kie-tooling-core/backend: 0.0.0 => 0.0.0-redhat-20210817
 - @kogito-tooling/build-env: 0.0.0 => 0.0.0-redhat-20210817 (private)
 - chrome-extension-pack-kogito-kie-editors: 0.0.0 => 0.0.0-redhat-20210817 (private)
 - @kie-tooling-core/chrome-extension: 0.0.0 => 0.0.0-redhat-20210817
 - @kogito-tooling/desktop: 0.0.0 => 0.0.0-redhat-20210817 (private)
 - @kogito-tooling/dmn-dev-sandbox-deployment-base-image: 0.0.0 => 0.0.0-redhat-20210817 (private)
 - @kogito-tooling/dmn-dev-sandbox-form-webapp: 0.0.0 => 0.0.0-redhat-20210817 (private)
```
so the dot is not allowed

**Review by Ginxo — CHANGES_REQUESTED** [review]
@radtriste I guess this is not possible due to technical limitations

**Comment by radtriste** [comment]
As it seems it cannot be harmonized due to Semantic version, we will handle the semantic version with a `-` on QE side.

**Pull request closed without merging**